.github/workflows/deploy-branch.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy-branch
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'rst/**.rst'
+jobs:
+  deploy-branch:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+      AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+      AWS_DEFAULT_REGION: ${{secrets.AWS_DEFAULT_REGION}}
+      S3_ENDPOINT_URL: ${{secrets.S3_ENDPOINT_URL}}
+      S3_UPLOAD_PATH: ${{secrets.S3_UPLOAD_PATH}}
+      TARANTOOL_UPDATE_KEY: ${{secrets.TARANTOOL_UPDATE_KEY}}
+      TARANTOOL_UPDATE_URL: ${{secrets.TARANTOOL_DEV_UPDATE_URL}}
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set branch name from source branch
+        run: echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+
+      - name: Start dev server deployment
+        uses: bobheadxi/deployments@v0.5.2
```

Comment:
Why do you use it, and not https://github.com/marketplace/actions/deployment-action?
(Describe it in PR comments, please)